game.py

```diff
@@ -0,0 +1,41 @@
+from exceptions import *
+from models import Player
+from models import Enemy
+import settings
+
+
+def play():
+    pl_name = input("Enter your name : ")
+    player = Player(pl_name)
+    level = 1
+    enemy = Enemy(1)
+
+    while True:
+        try:
+            player.attack(enemy)
+            player.defence(enemy)
+        except EnemyDown:
+            level += 1
+            player.score += 5
+            pass
+            del enemy
```

Comment:
not required to delete the enemy. You can delete this line and nothing will change.